CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+- Catch EoE service deinit error when disconnecting the drive.
```

Comment:
It is out of the scope, but add in the CHANGELOG.md virtual drive as a new feature, we forget it in the past

Reply:
Done!